src/page/managers/LoginManager.ts

```diff
@@ -36,8 +37,13 @@ export default class LoginManager {
 
     if (status && status >= 200 && status < 300) {
       Log.info("Successfully created user", result);
-    } else {
-      Log.error("Error creating user", result);
+    } else if (status && status >= 400 && status < 500) {
+      Log.error("Malformed request", result);
+    } else if (status && status >= 500) {
+      // retry indefinitely
+      Log.error("Server error. Retrying...");
+      await awaitableTimeout(1000);
```

Comment:
Probably want to set a longer await + backoff:

10 secs, 20, 30, 30, 30

Add max retry count: 5